djautotask/models.py

```diff
@@ -380,6 +380,7 @@ class Contact(TimeStampedModel):
     first_name = models.CharField(blank=True, null=True, max_length=20)
     last_name = models.CharField(blank=True, null=True, max_length=20)
     email_address = models.CharField(blank=True, null=True, max_length=50)
+    phone = models.CharField(blank=True, null=True, max_length=25)
```

Comment:
It looks like you have accidentally added an extra commit in this pull request. There is some extra stuff here about contacts. Please fix this branch and pull request to just the relevant changes.